modules/tinymce/changelog.txt

```diff
@@ -7,6 +7,7 @@ Version 5.4.0 (TBD)
     Added additional font related Oxide variables for secondary buttons allowing more specific styling. #TINY-6061
     Changed `advlist` toolbar buttons to only show a dropdown list if there's more than one option #TINY-3194
     Changed `mceInsertTable` command and `insertTable` API method to take optional header rows and columns arguments #TINY-6012
+    Fixed `autosave` isEmpty API aligned further with document functions #TINY-5953
```

Comment:
This isn't very clear, and an end-user isn't going to know what "document functions" mean. So maybe something like this instead?
```
Fixed the `autosave` isEmpty API incorrectly detecting non-empty content as empty #TINY-5953
```